examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.prm

```diff
@@ -11,9 +11,9 @@ subsection simulation control
   set method               = bdf1
   set number mesh adapt    = 0
   set output name          = result
-  set output frequency     = 50
+  set output frequency     = 5
   set startup time scaling = 0.6
-  set time end             = 5
+  set time end             = 0.01
```

Comment:
I don't think you wanted to modify the example no? At least, the example should still end after 5 seconds
Same for above and below

Reply:
Oh I didn't mean to commit that, that's a mistake